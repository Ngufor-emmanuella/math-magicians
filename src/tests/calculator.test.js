import React from 'react';
import renderer from 'react-test-renderer';
import Calc from '../components/calculator';

test('should match the snapshot', () => {
  const calc = renderer.create(<Calc />);
  const tree = calc.toJSON();
  expect(tree).toMatchSnapshot();
});
