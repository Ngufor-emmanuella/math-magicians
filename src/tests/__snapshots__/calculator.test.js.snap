// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<div
  className="general-container"
>
  <p
    className="general-paragraph"
  >
    Let us do some maths !
  </p>
  <div
    className="calculator-box"
  >
    <table
      cellSpacing="0"
      id="calculator"
    >
      <tbody
        id="calculator-body"
      >
        <tr
          className="result-box"
        >
          <td
            className="result"
            colSpan="4"
          >
            <span>
              0
            </span>
            <span />
            <span />
          </td>
        </tr>
        <tr>
          <td>
            <button
              className="no-color"
              onClick={[Function]}
              type="button"
            >
               AC 
            </button>
          </td>
          <td>
            <button
              className="no-color"
              onClick={[Function]}
              type="button"
            >
               +/- 
            </button>
          </td>
          <td>
            <button
              className="no-color"
              onClick={[Function]}
              type="button"
            >
               % 
            </button>
          </td>
          <td>
            <button
              className="color"
              onClick={[Function]}
              type="button"
            >
               รท 
            </button>
          </td>
        </tr>
        <tr>
          <td>
            <button
              className="no-color"
              onClick={[Function]}
              type="button"
            >
               7 
            </button>
          </td>
          <td>
            <button
              className="no-color"
              onClick={[Function]}
              type="button"
            >
               8 
            </button>
          </td>
          <td>
            <button
              className="no-color"
              onClick={[Function]}
              type="button"
            >
               9 
            </button>
          </td>
          <td>
            <button
              className="color"
              onClick={[Function]}
              type="button"
            >
               x 
            </button>
          </td>
        </tr>
        <tr>
          <td>
            <button
              className="no-color"
              onClick={[Function]}
              type="button"
            >
               4 
            </button>
          </td>
          <td>
            <button
              className="no-color"
              onClick={[Function]}
              type="button"
            >
               5 
            </button>
          </td>
          <td>
            <button
              className="no-color"
              onClick={[Function]}
              type="button"
            >
               6 
            </button>
          </td>
          <td>
            <button
              className="color"
              onClick={[Function]}
              type="button"
            >
               - 
            </button>
          </td>
        </tr>
        <tr>
          <td>
            <button
              className="no-color"
              onClick={[Function]}
              type="button"
            >
               1 
            </button>
          </td>
          <td>
            <button
              className="no-color"
              onClick={[Function]}
              type="button"
            >
               2 
            </button>
          </td>
          <td>
            <button
              className="no-color"
              onClick={[Function]}
              type="button"
            >
               3 
            </button>
          </td>
          <td>
            <button
              className="color"
              onClick={[Function]}
              type="button"
            >
               + 
            </button>
          </td>
        </tr>
        <tr>
          <td
            colSpan="2"
          >
            <button
              className="no-color"
              onClick={[Function]}
              type="button"
            >
               0 
            </button>
          </td>
          <td>
            <button
              className="no-color"
              onClick={[Function]}
              type="button"
            >
               . 
            </button>
          </td>
          <td>
            <button
              className="color"
              onClick={[Function]}
              type="button"
            >
               = 
            </button>
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
